### DISCUSSION TITLE
RAG: Generación Aumentada de Recuperación: Spanish Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#31] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/ES1
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=LX0YWd26dlU
Slides: https://github.com/microsoft/RAG_Hack/blob/main/slides/RAGHackSession-RAG101-Spanish.pptx

Please ask any follow-up questions here, and the speakers will see them.

**Comment by elbruno** [comment]
Hi, 
sharing some of the links that we used during the session:

- https://aka.ms/ragchatnet
- https://aka.ms/phi-3CookBook
- https://github.com/dotnet/eShopSupport

Best,
Bruno

**Comment by ppiova** [comment]
Hi All,
Here’s the repo that was used in one of the demos
- https://github.com/microsoft/chat-copilot

Pablito Piova
